README.md

```diff
@@ -1 +1,15 @@
-# docs
\ No newline at end of file
+# The Komiser Documentation
+
+This repository contains the files required to build the Komiser Documentation Hub. To any [contributors](https://github.com/komiserio/docs/blob/master/docs/contributing.md) willing for add to the project, before anything else, Thank you!
+
+## Contributing to the docs
+
+- Firstly, go ahead and **fork** the docs repository. Make any changes you want to your fork, and when you're ready to send those changes to us, go to your fork and create a new pull request.
+
+- If it takes longer than expected to get feedback from the `Oraculi` team, head over to the [Discord Server](https://discord.gg/Xz2Ru8qt) and **ping** an Oraculi staff member either in the `#general` or `#feedback` channel, unless you are a member of the private `contributor` channel. You can always request access to this channel. 
```

Comment:
could we replace this one with https://discord.oraculi.io so its easier for us to change the link if this one is depracted ?